api/envoy/config/rbac/v2alpha/rbac.proto

```diff
@@ -0,0 +1,172 @@
+syntax = "proto3";
+
+import "validate/validate.proto";
+import "envoy/api/v2/core/address.proto";
+
+package envoy.config.rbac.v2alpha;
+
+// Role Based Access Control (RBAC) provides service-level and method-level access control for a service.
+// The RBAC engine authorizes a request by evaluating the request context (expressed in the form of
+// :ref: `AttributeContext <envoy_api_msg_service.auth.v2alpha.AttributeContext>`) against the RBAC policies.
+//
+// RBAC policies are additive. The policies are examined in order. A request is allowed once a matching policy
+// is found (suppose the `action_type` is ALLOW).
+//
+// Here is an example of RBAC configuration. It has two policies:
+// * Service account "cluster.local/ns/default/sa/admin" has full access (empty permission entry means full access)
+//   to the service.
+// * Any user (empty principal entry means any user) can read ("GET") the service at paths with prefix "/products" or
+//   suffix "/reviews" when request header "version" set to either "v1" or "v2".
+//
+//   action_type: ALLOW
+//   policies:
+//     "service-admin":
+//       permissions:
+//       -
+//       principals:
+//         authenticated:
+//           name: "cluster.local/ns/default/sa/admin"
+//     "product-viewer":
+//       permissions:
+//       - paths: [prefix:"/products", suffix:"/reviews"]
```

Comment:
Nit: I think the right yaml syntax for this would be:
```yaml
    - paths: [prefix: "/products", suffix: "/reviews"]
```
or 
```yaml
    - paths: 
      - prefix: "/products"
      - suffix: "/reviews"
```
Note the whitespace between `:` and `"`. Without the whitespace it would be parsed as string instead of key/value map.

Reply:
Fixed. thanks.